Tests/SwiftDocCTests/Rendering/RenderBlockContent_ThematicBreakTests.swift

```diff
@@ -0,0 +1,35 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2024 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+
+@testable import SwiftDocC
+import Markdown
+import XCTest
+
+class RenderBlockContent_ThematicBreakTests: XCTestCase {
+    func testThematicBreakCodability() throws {
+        let thematicBreak = RenderBlockContent.thematicBreak
+        let data = try JSONEncoder().encode(thematicBreak)
+        let roundtrip = try JSONDecoder().decode(RenderBlockContent.self, from: data)
+        
+        guard case RenderBlockContent.thematicBreak = roundtrip else {
+            XCTFail("Didn't decode thematic break correctly.")
+            return
+        }
+        
+        XCTAssertEqual(thematicBreak, roundtrip)
```

Comment:
FYI: We have a `assertRoundTripCoding()` test helper for this. 
```suggestion
        assertRoundTripCoding(RenderBlockContent.thematicBreak)
```

Reply:
Wow this feels so much cleaner, thanks!